catalog/ui/catalog-ui-search/src/main/webapp/component/ingest-details/ingest-details.view.js

```diff
@@ -25,6 +25,7 @@ var UploadBatchModel = require('js/model/UploadBatch');
 var userInstance = require('component/singletons/user-instance');
```

Comment:
Let's try to decouple lists from this view.

Reply:
it is now decoupled